route.go

```diff
@@ -89,6 +91,26 @@ func (r *Route) GlobalMiddleware(middlewares ...httpcontract.Middleware) {
 	r.setMiddlewares(middlewares)
 }
 
+func HandleRecover(ctx httpcontract.Context, recoverCallback func(ctx context.Context, err any)) {
+	if err := recover(); err != nil {
+		if recoverCallback != nil {
+			recoverCallback(ctx, err)
+		} else {
+			ctx.Request().AbortWithStatusJson(http.StatusInternalServerError, gin.H{"error": "Internal Server Error"})
+		}
+	}
+}
+
+func (r *Route) Recover(callback func(ctx context.Context, err any)) {
```

Comment:
Sorry, mistake, should be: 

```suggestion
func (r *Route) Recover(callback func(ctx httpcontract.Context, err any)) {
```

Please modify https://github.com/goravel/framework/pull/723/files#diff-37b488415e77368d5640d20902b15e5603864dc8c5c1e715bc36e2a8510b81f8R19 synchronously.